src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.center {
+  position: absolute;
+  height: 5vmin;
+  width: 5vmin;
+  background-color: #f6a603;
+  border-radius: 50%;
+  z-index: 10;
+  margin: 0 auto;
+}
+
+.minutes {
+  position: absolute;
+  margin: auto 0;
+  bottom: 50%;
+  width: 3vmin;
+  height: 20vmin;
+  background-color: #0700ff;
+  z-index: 1;
+  transform-origin: bottom;
+  animation-name: move;
+  animation-duration: 3600s;
+  animation-iteration-count: infinite;
+  animation-timing-function: steps(60);
+}
+
+.seconds {
+  position: absolute;
+  margin: auto 0;
+  bottom: 50%;
+  width: 1.5vmin;
+  height: 38vmin;
+  background-color: #2c8000;
+  z-index: 1;
+  transform-origin: bottom;
+
+  animation-name: move;
+  animation-duration: 60s;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
```

Comment:
The 'seconds' class should be renamed to follow the BEM naming convention, for example 'stopwatch__seconds' if it's an element of the 'stopwatch' block.